src/Events/Custom_Tables/V1/WP_Query/Repository/Custom_Tables_Query_Filters.php

```diff
@@ -226,7 +226,12 @@ public function filter_posts_join( $join, WP_Query $query ) {
 			if ( ! in_array( $join_clause, $this->query_vars['join'], true ) ) {
 				$this->query_vars['join'][] = $join_clause;
 			}
-		} else if ( ! empty( $this->query_vars['join'] ) ) {
+		}
+
+		/*
+		 * @since TBD Changed from 'else if' to `if`.
+		 */
```

Comment:
Put the `@since` doc up above the method name, underneath the most recent `@since` doc.